hash/x1_cass.xml

```diff
@@ -296,6 +322,18 @@ Titles, publishers and release dates taken from:
 			</dataarea>
 		</part>
 	</software>
+	
+	<software name="machgjoe">
+		<description>Ginkou Goutou: Machinegun Joe vs. the Mafia</description>
```

Comment:
This should be "Ginkou Goutou - Machinegun Joe vs. the Mafia" to match the other descriptions.